out/notebooks/spl2/installer.ts

```diff
@@ -53,6 +53,7 @@ export async function installMissingSpl2Requirements(context: ExtensionContext,
             }
         } catch (err) {
             reject(`Error retrieving configuration '${configKeyJavaPath}', err: ${err}`);
+            return Promise.resolve();
```

Comment:
These new `Promise` statements fix an issue @JasonConger and I saw during a demo of mine where the installation process can get stuck in a loop - we were not actually exiting the function after resolving the promise for a bunch of cases.